src/Microsoft.ML.Core/Data/Repository.cs

```diff
@@ -432,6 +434,24 @@ public void Commit()
             Flush();
             Dispose(true);
         }
+
+        private static string GetProductVersion()
+        {
+            var assembly = typeof(RepositoryWriter).Assembly;
+
+            var assemblyInternationalVersionAttribute = assembly.CustomAttributes.FirstOrDefault(a =>
```

Comment:
>assemblyInternationalVersionAttribute [](start = 16, length = 37)

Nit: Can this be renamed `assemblyInformationalVersionAttribute`?

Reply:
OK sure